sample/sample.cpp

```diff
@@ -70,6 +70,30 @@ void configureToolbar(
     QAction* clearAction = toolbar.addAction("Clear");
     QAction* generateAction = toolbar.addAction("Generate");
     QAction* generateMultipleAction = toolbar.addAction("GenerateMultiple");
+    QAction* setRedBackgroundThread1 = toolbar.addAction("Set red background for thread 1");
+    QAction* clearRedBackgroundThread1 = toolbar.addAction("Clear red background");
+
+    setRedBackgroundThread1->connect(
+        setRedBackgroundThread1,
+        &QAction::triggered,
+        [&logView](bool) {
+            logView.setForegroundMapping(
+            "thread 1",
+            QBrush(Qt::red)
+            );
+        }
+        );
+
+    clearRedBackgroundThread1->connect(
+        clearRedBackgroundThread1,
+        &QAction::triggered,
+        [&logView](bool) {
+        logView.setForegroundMapping(
+            "thread 1",
+            std::nullopt
+        );
+        }
+    );
```

Comment:
I thought to allow the UI user to be able to set the color(s). This is the dev usage of the feature, IMO.

Reply:
Do you mean via context menus or something like that?

Reply:
Well, I think the context menu can be an option. At least for now. We need some kind of color picker for this. Maybe for now just give an action to set a color, which opens a dialog for inputting the logger name and the colors (foreground and background, both in hex code `#rgba` format). Note, we shouldn't forget about the initialization of the color fields. 

What do you think about this?